examples/megacli.cpp

```diff
@@ -3165,7 +3164,8 @@ autocomplete::ACN autocompleteSyntax()
     p->Add(exec_metamac, sequence(text("metamac"), localFSPath(), remoteFSPath(client, &cwd)));
     p->Add(exec_banner, sequence(text("banner"), either(text("get"), sequence(text("dismiss"), param("id")))));
 
-    p->Add(exec_notifydrive, sequence(text("notifydrive"), opt(wholenumber(60))));
+    p->Add(exec_drivenotificationson, sequence(text("drivenotificationson")));
+    p->Add(exec_drivenotificationsoff, sequence(text("drivenotificationsoff")));
```

Comment:
Probably nicer to have something like the existing `handles on|off` option :smiley:

Reply:
Updated.